docs/components/policy/obligations.md

```diff
@@ -0,0 +1,83 @@
+# Obligations
+
+Obligations are policy constructs that enable Policy Decision Point(PDP) - to - Policy Enforcement Point (PEP) directives that accompany a decision. They express additional controls the PEP should enforce—such as requiring MFA, applying watermarking, encrypting content, or limiting access by time. In effect, the PDP is saying “permit, provided these controls are enforced.” The PDP cannot compel or verify enforcement; it relies on the PEP to honor the obligations. 
+If the PEP cannot or will not enforce an obligation, it should decline to grant access.
+## Composition
+
+An obligation consists of:
+
+1. A Namespace
+2. A Definition
+3. Values
+4. Triggers
+
+Platform Policy Obligations can contain multiple Namespaces, each with multiple Definitions, and each Definition can have multiple Values. Each Value can have multiple Triggers. Each trigger can have an Action, Attribute Value, and PEP identifier.
+
+```mermaid
+graph LR;
```

Comment:
Can we add a case of a global obligation that is triggered by an action/attribute value but _no_ PEP? For example, an obligation to validate a specific assertion could be global across all PEPs.

Reply:
added with explanation and updated diagram example